packages/@sanity/portable-text-editor/src/editor/plugins/createWithPortableTextLists.ts

```diff
@@ -143,11 +143,14 @@ export function createWithPortableTextLists(types: PortableTextMemberSchemaTypes
       const selectedBlocks = [
         ...Editor.nodes(editor, {
           at: editor.selection,
-          match: (node) => editor.isListBlock(node) && node.listItem === listStyle,
+          match: (node) => editor.isTextBlock(node),
         }),
       ]
+
       if (selectedBlocks.length > 0) {
-        return true
+        return selectedBlocks.every(
```

Comment:
do we need to return `every` here or do we just care that there is `> 0` blocks that meet this condition?

Reply:
We do care that every block has the list style. For example if the selection was on the following:

> First block
>  - List second block
>  
> Third block

we'd have 3 `selectedBlocks`, but only 1 would have bullet list style, so we'd want to return false